Doc/glossary.rst

```diff
@@ -547,12 +547,10 @@ Glossary
       tasks such as compression or hashing.  Also, the GIL is always released
       when doing I/O.
 
-      Past efforts to create a "free-threaded" interpreter (one which locks
-      shared data at a much finer granularity) have not been successful
-      because performance suffered in the common single-processor case. It
-      is believed that overcoming this performance issue would make the
-      implementation much more complicated and therefore costlier to maintain.
-
+      As of Python 3.13, the GIL can be disabled using the ``--disable-gil``
+      build configuration or ``PYTHON_GIL=0`` environment variable. This feature
```

Comment:
This should link to https://docs.python.org/dev/using/configure.html#cmdoption-disable-gil and https://docs.python.org/dev/using/cmdline.html#envvar-PYTHON_GIL:

```suggestion
      As of Python 3.13, the GIL can be disabled using the :option:`--disable-gil`
      build configuration or :envvar:`PYTHON_GIL=0 <PYTHON_GIL>` environment variable. This feature
```

I think the text isn't quite right:

First Python must be configured and built with `--disable-gil`, _and_ then Python needs to be run with `PYTHON_GIL=0` (or `-X gil 0`).